src/libcrun/status.c

```diff
@@ -485,7 +486,20 @@ rmdirfd (const char *namedir, int fd, libcrun_error_t *err)
       ret = unlinkat (dirfd (d), de->d_name, 0);
       if (ret < 0)
         {
+retry_unlink:
```

Comment:
Not sure but indentation looks off here, could you do `clang-format` .

Reply:
Yes the CI fails for `clang-format` as well, could you please run `clang-format` and repush it. Thanks.

Reply:
Sorry, not used/didn't notice this style for goto labels, fixed.

Note `make clang-format` modifies quite a few other files, might want to do a "fix formatting" commit at some point